satpy/readers/seviri_l1b_hrit.py

```diff
@@ -455,6 +485,7 @@ def __init__(self, filename, filename_info, filetype_info,
         self._filename_info = filename_info
         self.ext_calib_coefs = ext_calib_coefs if ext_calib_coefs is not None else {}
         self.mda_max_array_size = mda_max_array_size
+        self.fill_HRV = fill_hrv
```

Comment:
PEP-8: Variable names should be lowercase

Reply:
True